tests/protos/worlds/template_proto_follow_solid_LUA.wbt

```diff
@@ -0,0 +1,66 @@
+#VRML_SIM R2021b utf8
```

Comment:
The name of this file doesn't seem to be consistent with others (LUA in capital letters).